src/geocat/viz/util.py

```diff
@@ -179,6 +179,74 @@ def add_right_hand_axis(ax,
     return axRHS
 
 
+def add_height_from_pressure_axis(ax,
+                                  heights=None,
+                                  pressure_units='hPa',
+                                  ticklabelsize=12,
+                                  label='Height (km)',
+                                  labelpad=10,
+                                  axislabelsize=16):
+    """Utility function that adds a right-hand Height axis to the plot, derived
+    from the left-hand Pressure axis.
+
+    Args:
+
+        ax (:class:`matplotlib.axes._subplots.AxesSubplot` or :class:`cartopy.mpl.geoaxes.GeoAxesSubplot`):
+            Current axes to the current figure
+        heights (:class:`numpy.ndarray` or list):
+            An array of desired height values in km.
+        pressure_units (:class:`str`):
+            The Pint-compliant unit string associated with the Pressure values. Assume to be hPa.
+        ticklabelsize (:class:`int`):
+            Text font size of tick labels. A default value of 12 is used if nothing is set.
+        label (:class:`str`):
+            Text to use for the right hand side label.
+        labelpad (:class:`float`):
+            Spacing in points from the axes bounding box. A default value of 10 is used if nothing is set.
+        axislabelsize (:class:`int`):
+            Text font size for y-axes. A default value of 16 is used if nothing is set.
+
+     Return:
+
+        axRHS (:class:`matplotlib.axes._subplots.AxesSubplot` or :class:`cartopy.mpl.geoaxes.GeoAxesSubplot`):
+            The created right-hand axis
```

Comment:
Not sure why, but the docs aren't picking up this part
<img width="816" alt="image" src="https://user-images.githubusercontent.com/38434768/234385320-e9f7957c-4ab1-48d7-b061-7ff29cc09c02.png">

Reply:
Hmm in other places I saw the docs doing weird things with return and return types. I will tackle this in the docstring overhaul.